### ISSUE TITLE
Configure black and isort only once in pyproject.toml

### ISSUE THREAD

**Issue by lukpueh** [theupdateframework/tuf#1684]
**Description of issue or feature request**:

#1626 added a `pyproject.toml`, which can be used to configure code formatters black and isort in one place only. Let's put the config parameters (line length, which files to include, etc) there.

See corresponding docs
- https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#configuration-via-a-file
- https://pycqa.github.io/isort/docs/configuration/config_files.html

**Current behavior**:
Repeated black and isort config in:
- [Contributor Docs](https://github.com/theupdateframework/python-tuf/blob/develop/docs/CONTRIBUTORS.rst#auto-formatting)
- [tox (CI)](https://github.com/theupdateframework/python-tuf/blob/bb15ecfb994290e723dd69fc728a2c7bd9201811/tox.ini#L44-L45)
- [Local .pre-commit hooks](https://github.com/theupdateframework/python-tuf/blob/be0cef067cecd566d60af96c15dc2e48bc4467af/.pre-commit-config.yaml#L11-L23) (NOTE: This is not part of the repo, but I have one locally, also see [contributor doc recommendation](https://github.com/theupdateframework/python-tuf/blame/develop/docs/CONTRIBUTORS.rst#L123-L125))
  
  

**Expected behavior**:
Single source of truth for black and isort config in `pyproject.toml`

**Comment by MVrachev** [comment]
I am planning to do that when resolving #1582.
I am submitting changes applying each of the linters separately and in the end, will add a pr moving the linters configuration options in `pyproject.toml` and actually starting linting the tests.

**Comment by MVrachev** [comment]
I was successful in moving all of the `black` configuration options in `pyproject.toml` except those about which files are included. These days I tried multiple ways to use the `include` option, so we can just call `black --config=pyproject.toml`, but I was not successful.
Finally, I found this comment https://github.com/psf/black/issues/861#issuecomment-680411125, and issue https://github.com/psf/black/pull/740 which explained that we can't use `black` without the `--path` argument included.
As a result, I will move all configuration options for black inside `pyproject.toml` without the actual directories that need to be linted.

**Comment by MVrachev** [comment]
It's the same situation with `isort` as it was with `black`.
I thought I can use only [src_path](https://pycqa.github.io/isort/docs/configuration/options.html#src-paths) and remove the target directories, but it seems I am wrong because when I tried running `isort` with `isort --settings-path=pyproject.toml` I got the error: `Error: arguments passed in without any paths or content.`
I am not the only one with that problem as a user explicitly asked about that https://github.com/PyCQA/isort/issues/1616 and I saw one project with source configuration https://github.com/Pylons/pyramid/blob/master/pyproject.toml, but they had to give explicit folders too https://github.com/Pylons/pyramid/blob/8061fce297cc7117d3e6e2b39e47512c7db2904f/tox.ini#L26 and https://github.com/Pylons/pyramid/blob/8061fce297cc7117d3e6e2b39e47512c7db2904f/tox.ini#L66

As a result, I will move all configuration options for `isort` inside pyproject.toml without the actual directories that need to be linted.

**Comment by MVrachev** [comment]
Sadly, I can't move `check` and `diff` attributes as well. In the documentation it's said that for both [check](https://pycqa.github.io/isort/docs/configuration/options.html#check) and [diff](https://pycqa.github.io/isort/docs/configuration/options.html#show-diff) are not supported in configuration files.
Additionally, in two issues https://github.com/PyCQA/isort/issues/954 and https://github.com/PyCQA/isort/issues/599  it was confirmed that in integration tests we should use `--check` and `--diff` the way we did until now.
